lms/envs/devstack.py

```diff
@@ -15,6 +15,7 @@
 SITE_NAME = 'localhost:8000'
 # By default don't use a worker, execute tasks as if they were local functions
 CELERY_ALWAYS_EAGER = True
+HTTPS = 'off'
```

Comment:
Not a big deal, but in the future, it's usually easier to wait until we merge the feature to our remote before opening a PR upstream. That way we can address any/all issues with our internal review before potentially receiving duplicate feedback from an upstream reviewer.

This approach saves you and the reviewers time. Additionally, we do want this feature merged to origin soon-ish, but we aren't in any real rush to push it upstream. We're doing that as a "favor" as good community members, but we don't really care if it takes a long time to merge there.

Also, when you open a PR upstream, please tag the Stanford reviewer who looked at the code on our end. That way, we can weigh in on feedback that may be provided from edX.